packages/edit-widgets/src/components/header/document-tools/index.js

```diff
@@ -75,14 +72,10 @@ function DocumentTools() {
 		[ setIsListViewOpened, isListViewOpen ]
 	);
 
-	// If there's a block toolbar to be focused, disable the focus shortcut for the document toolbar.
-	const { showFixedToolbar } = useShowBlockTools();
-
 	return (
 		<NavigableToolbar
 			className="edit-widgets-header-toolbar"
 			aria-label={ __( 'Document tools' ) }
-			shouldUseKeyboardFocusShortcut={ ! showFixedToolbar }
```

Comment:
Prevents rerender on widgets editor document tools when block is selected. Shortcut works as intended without this prop.